### PR TITLE
feat: SDA-1553 Add screen-sharing-frame, for sharing screen

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
## Description
A few sentences describing the overall goals of the pull request's commits. 
Describe the problem or feature in addition to a link to the 
[JIRA-ticket](https://perzoinc.atlassian.net/browse/JIRA-ticket)

## Solution Approach
Describe the approach you've taken to implement this change / resolve the issue

## Related PRs
List related PRs against other branches/repositories:

branch | PR
------ | ------
other_pr_dev | [link]()

### CODE REVIEW

File: src/app/window-handler.ts [hunk 1]
```diff
@@ -699,6 +704,20 @@ export class WindowHandler {
             });
         }
 
+        if (displayId !== '') {
+            const displays = electron.screen.getAllDisplays();
+
+            displays.forEach((element) => {
+                if (displayId === element.id.toString()) {
+                    this.createScrenSharingFrameWindow('screen-sharing-frame',
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: @johankwarnmarksymphony JIRA also includes highlighting of the application but that's not working here

File: src/renderer/components/screen-sharing-frame.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+     */
+    public render(): JSX.Element {
+
+        return (
+            <div className={classNames('ScreenSharingFrame')}>
+            </div>
+        );
+    }
+
+    /*public componentDidMount(): void {
```
Comment (KiranNiranjan) [inline src/renderer/components/screen-sharing-frame.tsx]: Please remove commented code

Comment (johankwarnmarksymphony) [reply src/renderer/components/screen-sharing-frame.tsx]: Fixed

File: src/app/window-handler.ts (hunk 1 above)
Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: @KiranNiranjan I don't understand what you mean.
I loop over all displays and if Id match I draw the frame around that display

Comment (KiranNiranjan) [reply src/app/window-handler.ts]: Displays is working but I mean highlighting the applications is not

Comment (johankwarnmarksymphony) [reply src/app/window-handler.ts]: True, I will talk to the designer, if we should wait until we have fixed the highlighting of the application. Or split into two tickets 
1) Frame around sharing desktop
2) Frame around sharing application

**Pull request merged**